build_luajit.sh

```diff
@@ -17,6 +17,10 @@ else
     if [ ! -d "luajit_repo" ]; then 
       git clone -b v2.1 https://github.com/LuaJIT/LuaJIT luajit_repo
     fi
+    
+    if [ ! -d "ratorjit_repo" ]; then 
+      git clone https://github.com/raptorjit/raptorjit ratorjit_repo
```

Comment:
Should we be using a fixed tag/hash for our VMs?

Reply:
Good point -- yes, we should.

Tom: what Edd means is that by fixing a hash, we know precisely which version of each JIT we're benchmarking.

That said, we could allow people to specific an option that means "just download the latest versions".

Reply:
I added locking to a commit hash